core/modules/widgets/void.js

```diff
@@ -0,0 +1,49 @@
+/*\
+title: $:/core/modules/widgets/Void.js
+type: application/javascript
+module-type: widget
+
+Void widget that is not intended for render. It still renders all its children.
```

Comment:
Could we avoid introducing a new widget by extending the existing genesis widget to have some combination(s) of attributes that have the effect of generating the functionality of the void widget.